packages/itwinui-css/package.json

```diff
@@ -3,19 +3,16 @@
   "version": "0.63.1",
   "author": "Bentley Systems",
   "license": "MIT",
-  "main": "src/index.scss",
+  "main": "css/all.css",
   "files": [
     "css",
-    "scss",
     "CHANGELOG.md",
     "LICENSE.md"
   ],
-  "description": "CSS (Sass/SCSS) library for building beautiful and well working web UI components within Bentley Systems & iTwin.js applications.",
+  "description": "CSS library for building beautiful and well working web UI components within Bentley Systems & iTwin.js applications.",
   "homepage": "https://github.com/iTwin/iTwinUI",
   "keywords": [
     "css",
-    "sass",
-    "scss",
```

Comment:
Maybe remove it from `.gitignore` too.

Reply:
I kept it in the gitignore for now because otherwise everyone will have a bunch of untracked scss files when switching branches